### PR TITLE
chore: adjust workflow

### PR DESCRIPTION
**Pull request by uziassantosferreira**
### Description and Example

Fix workflow problems

### Checklist

Please, check if these important points are met using `[x]`:

- [X] I read the [PR Guide] and followed the process outlined there for submitting this PR.
- [X] I avoided _breaking changes_ by not changing public APIs that people rely on. 
- [X] I am willing to follow-up on review comments in a timely manner.
- [X] I have made the documentation changes or I created an issue explaining how to document this change on [Docs issues](https://github.com/ZupIT/beagle-docs/issues). Please link the issue here:

[PR Guide]: https://github.com/ZupIT/beagle-android/blob/main/doc/contributing/pull_requests.md

### CODE REVIEW

**Comment by uziassantosferreira** [comment]
@hectorcustodiozup I think this is the best workflow because the unit test it's important to run in the PR I just skip the sonarqube and this will be run when merging PR in the master.

File: .github/workflows/horusec-pr.yml [hunk 1]
```diff
@@ -31,16 +29,6 @@ jobs:
...
-                  aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
-                  aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-                  aws-region: us-east-1
-                  secrets: |
-                      beagle/core/security
-                  parse-json: true
-
             - name: Running Horusec
               run: |
                   curl -fsSL https://raw.githubusercontent.com/ZupIT/horusec/master/deployments/scripts/install.sh | bash -s latest
```
Comment (hectorcustodiozup) [inline .github/workflows/horusec-pr.yml]:
```suggestion
                  curl -fsSL https://raw.githubusercontent.com/ZupIT/horusec/main/deployments/scripts/install.sh | bash -s latest
```

File: .github/workflows/horusec-pr.yml [hunk 2]
```diff
@@ -20,8 +20,6 @@ on:
     pull_request:
         branches:
             - main
-            - horusec
-            - release/*
```
Comment (hectorcustodiozup) [inline .github/workflows/horusec-pr.yml:24]: We should keep SAST validation on release branches according to our best practices validations.

**Review by hectorcustodiozup — CHANGES_REQUESTED** [review]
Everything should be ok to be merged, just added a couple tweaks.

**Review by hectorcustodiozup — APPROVED** [review]

**Pull request merged**